e2e/transform/transform-esm-runner/runner.mjs

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import throat from 'throat';
+import testResult from '@jest/test-result';
+
+const {createEmptyTestResult} = testResult;
```

Comment:
No.
Because ts will compile to common js modules, and export using `Object.defineProperty` with getter.
ESM named import from CJS module only support `exports.name = value` way. [esm commonjs namespaces](https://nodejs.org/docs/latest-v14.x/api/esm.html#esm_commonjs_namespaces)

[CI error log](https://app.circleci.com/pipelines/github/facebook/jest/4540/workflows/ca14370c-5106-4eac-942a-93a027e2f6dc/jobs/103843/tests)

Reply:
Tested. It worked.